### PR TITLE
Add datastore to use persistent storage.

### PR DESCRIPTION
**Pull request by Daljin**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -56,9 +59,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    String message = request.getParameter("messageInput");
+    String email = request.getParameter("emailInput");
 
-    messages.add(message);
+    // Create an entity with a kind of Comment. 
+    Entity commentEntity = new Entity("Comment");
+    
+    // Add properties to the entity: email and message.
+    commentEntity.setProperty("emailInput", email);
+    commentEntity.setProperty("messageInput", message);
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You can move the persistence part to a method. It makes the code more readable,

doPost will be something like:
persistMessage(request.getParameter("emailInput"), request.getPara,eter("essageInput");

response.sendRedirect("index.html");

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -56,9 +59,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String message = request.getParameter("messageInput");
+    
+    // Get the input from the form.
+    String message = request.getParameter("messageInput");
+    String email = request.getParameter("emailInput");
 
-    messages.add(message);
+    // Create an entity with a kind of Comment.
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You can remove this comment. The code makes the intent clear.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (Daljin) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (Daljin) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove and done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -54,12 +57,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
     // Redirect the url to index.html.
     response.sendRedirect("index.html");
   }
+
+  // Get the input from the form, and sends them to the datastore.
+  public void persistMessage(String email, String message) {
+    Entity commentEntity = new Entity("Comment");
+    
+    // Add properties to the entity: email and message.
+    commentEntity.setProperty("emailInput", email);
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: you can remove this comment too. The code is pretty self explanatory.

**Pull request merged**